server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestRefreshAction.java

```diff
@@ -51,6 +51,11 @@ public String getName() {
         return "refresh_action";
     }
 
+    @Override
+    public boolean allowSystemIndexAccessByDefault() {
+        return true;
```

Comment:
Do you recall why we need to allow system indices to be refreshed?

Reply:
I don't think we do - I had intended to remove this.